_visualisation/core/focus-depth.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Focus and Depth
+position: 1.06
+---
+
+Normally, diagrams will focus on a given Entity and show all Parties with either ownership or control interests in that Entity, either directly or indirectly. However, you may wish to adopt a different focus or limit which parties or interests are shown.
+{: .lead}
+
+
+## Focus
+
+Your diagram's focus determines which Parties and interests are included.
+
+### Parties
+
+**Diagrams have either an Object or Subject Focus.** That is, they either:
+
+* Focus on selected Objects (at the ends of Chains), then trace backwards all of their Beneficiaries.
+* Focus on selected Subjects (at the starts of Chains), and trace forwards all the Objects they are interested in.
+
+#### Object Focus
+
+![BOVS Diagram with Object Focus](/visualisation/diagrams/bovs-core-focus-object.png){: style="width: 87.5%" }
+
+Diagrams with an Object Focus (the most common) show all the Parties that exist in the reverse direction of the Arrows from the selected Objects (usually just one). For instance, it may trace all of the Parties interested in a specific Company, back to a set of Persons.
+
+#### Subject Focus
+
+![BOVS Diagram with Subject Focus](/visualisation/diagrams/bovs-core-focus-subject.png){: style="width: 87.5%" }
+
+Diagrams with a Subject Focus show all the Parties that exist in the forward direction of the Arrows from the selected Subjects (usually just one). For instance, it may show all the interests of a specific Person.
+
+### Interests
+
+**Normally, BOVS diagrams show both ownership and control interests,** that is, they lack an Interest Focus.
+
+#### Interest Focus
+
+![BOVS Diagram with Interest Focus](/visualisation/diagrams/bovs-core-focus-interest.png){: style="width: 87.5%" }
```

Comment:
Consider highlighting the 'Parties Controlling Company F' label of the diagram more, and or annotating to show that this label is required in cases where a diagram has an interest focus.